docs_devel/HowToCreateFilterPlugin.md

```diff
@@ -32,21 +34,10 @@ left to right as described in the table below:
 | Version     | Plugin-Version, Bundle-Version, Implementation-Version |
 | Author      | Plugin-Author, Implementation-Vendor, Built-By         |
 | Description | Plugin-Description                                     |
+| Link        | Plugin-Link                                            |
+| License     | Plugin-License                                         |
+| Category    | Plugin-Category                                        |
 
-### plugins for OmegaT 2.1.3 and up
-A plugin should be declared in `META-INF/MANIFEST.MF`:
-
-    OmegaT-Plugin: true
```

Comment:
Any reason to delete this? If authors want to target old versions of OmegaT (not sure why, but maybe there is a reason) then they will probably want to know this.

Reply:
Does OmegaT team still support OmegaT < 3.0?
If author want to target old versions, author can see old document from old source.